config/initializers/field_with_errors.rb

```diff
@@ -1,8 +1,10 @@
-ActionView::Base.field_error_proc = proc do |html_tag, instance|
-  if html_tag =~ /type="radio"/
+# frozen_string_literal: true
+
+ActionView::Base.field_error_proc = proc do |html_tag, _instance|
+  if html_tag.include?('type="radio"') || html_tag.include?('data-skip-error-class="true"')
     html_tag.html_safe # Do not wrap radio buttons
   else
     # Wrap other inputs with the default div
     %(<div class="field_with_errors">#{html_tag}</div>).html_safe
```

Comment:
![Screenshot 2025-03-04 at 9 31 17 PM](https://github.com/user-attachments/assets/54463337-619b-4c8c-87f3-4eacd893f33f)
this is the only other UI issue I've noticed - the `<div class="field_with_errors">` doesn't take the full width and it looks weird next to the normal inputs (those are styled with 100% width). adding the `width-full` class seems to fix it afaict.
```suggestion
    # Wrap other inputs with the default div
    %(<div class="field_with_errors width-full">#{html_tag}</div>).html_safe
```

Reply:
(FYI this change only takes affect after restarting the server)

Reply:
Yeah this is why I tried to add a data attribute to skip this rails feature entirely. Must have not entirely worked. The width-full thing might be ok

Reply:
oh shoot, you're right. I should have taken my own advice and restarted the server before testing this. I tried it again after restart and it's working correctly. 🎬